content/updates/2020-09/1-flat-files-exports-with-labels.md

```diff
@@ -0,0 +1,31 @@
+---
+pim_announcement_img: ./img/Product-export-profile.png
+pim_announcement_alt_img: "Flat files exports with labels"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Export flat files with labels
+::: meta-data type="New" features="Collaboration, Productivity" available="mid September" link-to-doc="../articles/exports.html"
+:::
+
+To save you time, it is now possible to export your product data in flat files with labels and without codes.
+This improvement has been added to both existing features:
+* Quick export
+* Product export profile
+
+You will notice an optional button named `With label`, helping you to export your product data with labels.
+When activated, the **headers** (properties & attributes) and the **values** in your flat file will be displayed in the locale you will choose instead of displaying their codes.
```

Comment:
The word "displayed" sounds really weird here. Do we agree that the feature aims at **exporting** those labels? Or maybe say something like
```suggestion
When activated, the **headers** (properties & attributes) and the **values** in your exported flat file will be displayed in the locale you will choose instead of displaying their codes.
```